sympy/core/relational.py

```diff
@@ -1250,7 +1250,7 @@ def is_neq(lhs, rhs):
     return fuzzy_not(is_eq(lhs, rhs))
 
 
-def is_eq(lhs, rhs):
+def is_eq(lhs, rhs, assumptions=True):
```

Comment:
I think that `assumptions=True` seems somewhat confusing here. I would expect that to mean something like "use the assumption system" when it fact it means "do not use any assumptions apart from the code assumptions".

Reply:
I made `None` as default assumption.